components/table/table-wrapper.js

```diff
@@ -249,7 +249,13 @@ export const tableStyles = css`
 
 	/* sticky + scroll-wrapper */
 	d2l-table-wrapper[sticky-headers][sticky-headers-scroll-wrapper] .d2l-table {
-		display: block;
+		display: flex;
+		flex-direction: column;
+	}
+
+	d2l-table-wrapper[sticky-headers][sticky-headers-scroll-wrapper][_no-scroll-width] .d2l-table > thead,
+	d2l-table-wrapper[sticky-headers][sticky-headers-scroll-wrapper][_no-scroll-width] .d2l-table > tbody {
+		display: table;
```

Comment:
Might be worth a bit if screen reader testing. In the past, applying table CSS could actually change the semantics in certain browsers / screen readers, even though we'd expect it to be purely stylistic. I think it was inconsistent across browsers/screen readers. They might have corrected that by now, but I remember one of the key W3C ally people noting it only a few years ago.